src/components/masters/program/sidebar/Sidebar.jsx

```diff
@@ -62,4 +72,19 @@ const Sidebar = () => (
   </>
 );
 
+Sidebar.defaultProps = {
+  programDocuments: null,
+};
+
+Sidebar.propTypes = {
+  programDocuments: (PropTypes.shape({
```

Comment:
nit: the wrapping `(` and `)` around `PropTypes` is unnecessary

Reply:
added 👍